rust/crd/src/lib.rs

```diff
@@ -238,6 +238,10 @@ pub struct CommonNodeConfig {
     /// Time period Pods have to gracefully shut down, e.g. `30m`, `1h` or `2d`. Consult the operator documentation for details.
     #[fragment_attrs(serde(default))]
     pub graceful_shutdown_timeout: Option<Duration>,
```

Comment:
nit: Please spread them apart according to style guide
```suggestion
    pub graceful_shutdown_timeout: Option<Duration>,

```